@kiva/kv-components/vue/KvLightbox.vue

```diff
@@ -200,84 +207,97 @@ export default {
 			default: false,
 		},
 	},
-	data() {
-		return {
-			mdiClose,
-			makePageInert() {}, // reference to aria-hide function
-		};
-	},
-	computed: {
-		role() {
-			if (this.variant === 'alert') {
+	emits: [
+		'lightbox-closed',
+	],
+	setup(props, { emit }) {
+		const {
+			visible,
+			variant,
+			preventClose,
+		} = toRefs(props);
+
+		const kvLightboxBody = ref(null);
+		const controlsRef = ref(null);
```

Comment:
I think these refs will need to be bound in the template. For instance, `ref="kvLightbox"` would change to `:ref="kvLightbox"`

Reply:
I though the same, but, 
https://v3.vuejs.org/guide/composition-api-template-refs.html

Your kvbutton has the same.

Reply:
oh, right, good point!